index.html

```diff
@@ -234,8 +243,11 @@ <h3>BrowserCaptureMediaStreamTrack</h3>
           <dd>
             <p>
               Calls to this method instruct the user agent to start/stop cropping a [=self-capture
-              video track=] to the contours of the element referenced by
-              <var>cropTarget</var>. Whenever {{BrowserCaptureMediaStreamTrack/cropTo}} is invoked,
+              video track=] to the <a href="https://drafts.csswg.org/cssom-view/#dom-element-getboundingclientrect">
+              bounding client rectangle</a> of <var>cropTarget</var>.[[\Element]].
+              Since the track is restricted to the visible viewport of the [=display-surface=],
+              the captured area will be the intersection of the visible viewport and the element bounding client rectangle.
+              Whenever {{BrowserCaptureMediaStreamTrack/cropTo}} is invoked,
```

Comment:
I think this the core of the PR, according to the context set by the attached issue. I'd move everything else to separate PRs and discuss those changes separately.

I'd gladly merge this PR if:
1. We leave just this paragraph in the PR (lines 236 and onwards).
2. We remove reference to `<var>cropTarget</var>.[[\Element]]` and replaced it with `Let <var>element</var> be the {{HTMLElement}} for which cropTarget was produced. Then...`

(Btw, HTMLElement rather than Element, until we settle issue #13. I'm OK with Element personally, but let's wait for @alvestrand to elaborate his thinking on this topic.

Reply:
Let's make progress the regular way. I just spent a lot of time commenting on each thread here to make progress.

Reply:
Goal of this PR is: introduce the slot, use it in one place to define more precisely cropping. We can split the PR in two.
In the interest of time, I would tend to merge this PR, merge the serialisation PR on top of this one reusing the weak reference slot.